### ISSUE TITLE
Doesn't run on free HyperV Server 2012 R2 operating system

### ISSUE THREAD

**Issue by trackroute** [nicehash/NiceHashMiner2#14]
NiceHashMiner version 1 runs perfectly on the free Microsoft HyperV Server 2012 R2 operating system. 

The new version (V2) crashes on the free Microsoft HyperV Server 2012 R2 operating system because dxgi.dll is missing. 

Specifically, from a business perspective:
- In order to keep costs down I run the free HyperV Server 2012 R2 operating system on each mining rig, this saves the cost of a Windows license per rig. 
- The decision to require DirectX (or whatever dxgi.dll is) increases the cost per rig significantly because I now have to purchase a full Windows operating system license for each rig. 
- Having to spend $100-$199 per rig just because I need DirectX to render a simple GUI is to me a complete waste of money.

In summary – Do you seriously have to use DirectX to render the GUI (or whatever dxgi.dll is used for)?  Can’t you just stick to the basic dlls so that the app will remain compatible with the free Microsoft HyperV Server operating system?

**Comment by jaretburkett** [comment]
You should be able to copy dxgi.dll from a desktop version of Windows located in `/Windows/System32` to the same folder on your rigs. It may crash due to additional DLLs, but you should be able to do the same with those. We will look for a better work around, but for now, give that a shot and let me know if it works.
